DoxyGen/src/cmsis_freertos.txt

```diff
@@ -792,7 +792,7 @@ Additionally, please take a look at the following:
 
 Overview of all CMSIS-RTOS C API v2 functions that are implemented in CMSIS-FreeRTOS.
 
-<a href="http://arm-software.github.io/CMSIS_5/RTOS2/html/group__CMSIS__RTOS__KernelCtrl.html">Kernel Information and Control</a>
+<a href="https://arm-software.github.io/CMSIS_5/RTOS2/html/group__CMSIS__RTOS__KernelCtrl.html">Kernel Information and Control</a>
```

Comment:
@JonatanAntoni Doxygen doesn't create link out of this. I suspect that's a bug in Doxygen. Did anyone saw something like this before? Do we have a workaround?

(applies to all similar items below)